daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTest.scala

```diff
@@ -238,6 +240,13 @@ object SpeedyTest {
     }
   }
 
+  @SuppressWarnings(Array("org.wartremover.warts.Any"))
```

Comment:
This is also defined in [`SBuiltinTest`](https://github.com/digital-asset/daml/pull/3416/files#diff-ae9014deee9be354b39ebd26faa0150bR1231). How about moving this to `Equality`'s companion object?